### ISSUE TITLE
Doesn't support DXTn raw data.

### ISSUE THREAD

**Issue by zhangnan0324** [dlemstra/Magick.NET#374]
### Prerequisites

- [x] I have written a descriptive issue title
- [x] I have verified that I am using the latest version of Magick.NET
- [x] I have searched [open](https://github.com/dlemstra/Magick.NET/issues) and [closed](https://github.com/dlemstra/Magick.NET/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description

This is a great project, but it seems to be an error when exporting images in DXTn format. The DDS format works fine, but it might be better to get DXTn raw data.  
### Steps to Reproduce

      using (MagickImage image = new MagickImage("1.jpg"))
        {
            image.Format = MagickFormat.Dxt1;
            var bytes = image.ToByteArray();
        }
### System Configuration

- Magick.NET version:
Magick.NET-Q8-x64.7.10.1
- Environment (Operating system, version and so on):
Windows 7, Visual Studio 2017，.Net Core 2.2
- Additional information:

**Comment by dlemstra** [comment]
It appears that you found a bug and I just pushed a patch to the ImageMagick repository to resolve this. I will try to publish a new release this weekend to resolve this.

**Comment by dlemstra** [comment]
I tried to add some unit tests but I could not get them to fail. After a while I realized this was happening because another method call was made. And that means you could use the following as a temporary work around:
```C#
MagickNET.GetFormatInformation(MagickFormat.Dxt1);
```

**Comment by zhangnan0324** [comment]
Thank you for your work, I tried it as you said, there is really no error, but it returns the data in DDS format.

**Comment by dlemstra** [comment]
Can you clarify what you mean? `Dxt1` and `Dxt5` are compression method. Do you want the image without the dds header?

**Comment by zhangnan0324** [comment]
Yes, I use it in Unity3d, there is a method to load DXTn raw data directly to load the image quickly.
Public void LoadRawTextureData(byte [] data);
Public void LoadRawTextureData(IntPtr data,int size);
Of course, I can remove the title myself, but from a performance perspective, it might be better if I never generate a title.

**Comment by zhangnan0324** [comment]
Just like MagickFormat.Rgb.

**Comment by dlemstra** [comment]
I think it might be possible to add an option for this to the DDS encoder. Will try to take a look at it this weekend.
